src/main/kotlin/ro/hibyte/betting/service/UserGroupService.kt

```diff
@@ -0,0 +1,54 @@
+package ro.hibyte.betting.service
+
+import org.springframework.stereotype.Service
+import org.springframework.web.multipart.MultipartFile
+import ro.hibyte.betting.dto.UserGroupDto
+import ro.hibyte.betting.entity.UserGroup
+import ro.hibyte.betting.entity.UserProfile
+import ro.hibyte.betting.repository.UserGroupRepository
+import ro.hibyte.betting.repository.UserProfileRepository
+
+@Service
+class UserGroupService(
+    private val userGroupRepository: UserGroupRepository,
+    private val waspService: WaspService,
+    private val userProfileRepository: UserProfileRepository
+) {
+    fun getAll(): List<UserGroup> = userGroupRepository.findAll()
```

Comment:
As a code formatting rule, try to always leave an empty line above each method - in this case, add a new line above methods `getAll`, `getOne` and `delete`